activity/index.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html>
+
+<head>
+    <title>Community Activity</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.1/Chart.bundle.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```

Comment:
Is jquery 3 not compatible with Chart.js?

Reply:
Wasn't using it because it fails silently on `getJSON` errors(and I need logging), will change.

Reply:
I belive it's `.fail()` in jQuery 3

Reply:
yep, changing.